start.sh

```diff
@@ -34,8 +34,25 @@ if [ $? -ne 0 ]; then
     exit
 fi
 
-echo Building asssets...
-webpack
+if [ $1 = "--setup-devenv" ] || [ $2 = "--setup-devenv" ]; then
```

Comment:
I didn't know this, but `=` and `==` are aliases. http://tldp.org/LDP/abs/html/comparison-ops.html

From testing it seems to be so.